### ISSUE TITLE
DB file should be moved into a dedicated directory

### ISSUE THREAD

**Issue by ayaka14732** [yixiaoer/schedule-main#2]
Currently the DB file is mistakenly saved in `~/.cache/data.json`. It should be saved in `~/.cache/schedule-main/data.json` instead.

**Comment by ayaka14732** [comment]
Sample code:

```python
from pathlib import Path

home_directory = Path.home()
db_file_path = home_directory / '.cache' / 'schedule-main' / 'data.json'
db_file_path.parent.mkdir(parents=True, exist_ok=True)

with open(db_file_path, 'w') as file:
    ...
```

This ensures that the solution is platform-agnostic.

**Comment by ayaka14732** [comment]
Fixed by #3
